lib/client-assets.php

```diff
@@ -1093,6 +1093,7 @@ function gutenberg_editor_scripts_and_styles( $hook ) {
 		'bodyPlaceholder'     => apply_filters( 'write_your_story', __( 'Write your story', 'gutenberg' ), $post ),
 		'isRTL'               => is_rtl(),
 		'autosaveInterval'    => 10,
+		'wpAdminURL'          => admin_url(),
```

Comment:
Another challenge is that `admin_url()` is filterable, meaning it can be modified based on the provided path. I don't think this is a significant issue, but something to be aware of in the decision-making process.

It appears `getWPAdminURL()` is only used in a few places:

```
editor/utils/url.js
14:	return getWPAdminURL( 'post.php', { post: postId, action: 'edit' } );
25:export function getWPAdminURL( page, query ) {

editor/components/post-permalink/index.js
19:import { getWPAdminURL } from '../../utils/url';
112:						href={ getWPAdminURL( 'options-permalink.php' ) }

editor/components/post-last-revision/index.js
13:import { getWPAdminURL } from '../../utils/url';
19:				href={ getWPAdminURL( 'revision.php', { revision: lastRevisionId, gutenberg: true } ) }

editor/store/effects.js
27:import { getPostEditUrl, getWPAdminURL } from '../utils/url';
273:		window.location.href = getWPAdminURL( 'edit.php', {
```

Can we expose these individual URLs instead, and remove `getWPAdminURL()` entirely?

Reply:
Exposing the URLs could be worthwhile, especially if it abstract a level higher. 

Because even with this fix it doesn't really solve the issue I'm trying to address, it fixes a bug and prevents some weird behavior after "Move to Trash" is called but what we really want is to define where the user is redirected. After trash event it is hard coded to edit.php which doesn't make that much sense in our P2/front-end use case.

I'm not sure how granular, or annoying, it might be to set all of these, for this example, something like "RedirectURLAfterTrash()"

Reply:
Fair enough. Want to open a new issue for future consideration?

Reply:
@danielbachhuber Created => https://github.com/WordPress/gutenberg/issues/7136